prow/plugins/branchcleaner/branchcleaner.go

```diff
@@ -0,0 +1,64 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package branchcleaner
+
+import (
+	"fmt"
+
+	"github.com/sirupsen/logrus"
+
+	"k8s.io/test-infra/prow/github"
+	"k8s.io/test-infra/prow/pluginhelp"
+	"k8s.io/test-infra/prow/plugins"
+)
+
+const (
+	pluginName = "branchcleaner"
+)
+
+func init() {
+	plugins.RegisterPullRequestHandler(pluginName, handlePullRequest, helpProvider)
+}
+
+func helpProvider(config *plugins.Configuration, enabledRepos []string) (*pluginhelp.PluginHelp, error) {
+	return &pluginhelp.PluginHelp{
+		Description: "The branchcleaner automatically deletes the source branch of Pull Requests after it got successfully merged if it is on the same repo as the target branch. This is helpful to keep repos that don't allow forking clean."}, nil
```

Comment:
Maybe

```
The branchcleaner plugin automatically deletes source branches for merged PRs between two branches on one repository.
```

Reply:
I have slightly adjusted your suggestion to "The branchcleaner plugin automatically deletes source branches for merged PRs between two branches on the same repository"